sfbrigade/datasci-earthquake
# Integrating streamlit app concept

# Description

Updated files in streamlit_app including app.py, the_map.py, requirements.txt within streamlit_app directory, and data files. If you do not want to mess with the existing streamlit_app code it can be merged into a new branch (e.g. KG/dataexplore). Only files in streamlit_app should be impacted by a merge. 

## Type of changes
- [x ] New Feature

## Testing
I did not add automated tests

## How to test

-Install requirements.txt
-From streamlit_app/archive/streamlit_app_concept, run streamlit run app.py
-Enter an address into the search bar, press "enter", see whether information is provided and accurate
